cmd/eval.go

```diff
@@ -408,6 +417,17 @@ func setupEval(args []string, params evalCommandParams) (*evalContext, error) {
 		regoArgs = append(regoArgs, rego.ParsedInput(inputValue))
 	}
 
+	schemaBytes, err := readSchemaBytes(params)
+	if err != nil {
+		return nil, err
+	} else if schemaBytes != nil {
+		schema, err := ast.CompileSchemas(schemaBytes, nil)
```

Comment:
Per comment in ast/compile.go, let's hide this as an implementation detail of the compiler's type checking stage. Just pass the raw schemas thru to the compiler.

Reply:
Done